cypress/support/step-definitions/select-steps.js

```diff
@@ -0,0 +1,71 @@
+import { select, selectInput, selectPill } from '../../locators/select';
+
+Then('Select typeAhead is disabled', () => {
+  select()
+    .should('have.attr', 'placeholder', 'Please Select...');
```

Comment:
Here you break line of select.should() but below not. Please unify and break only if line is too long

Reply:
unified